qt_ui/windows/basemenu/airfield/QAircraftRecruitmentMenu.py

```diff
@@ -91,8 +91,13 @@ def __init__(self, game_model: GameModel, control_point: ControlPoint) -> None:
         self.setAlignment(Qt.AlignLeft)
 
     def update_label(self) -> None:
-        next_turn = self.control_point.allocated_aircraft()
-        max_amount = self.control_point.total_aircraft_parking
+        parking_type = ParkingType()
+        parking_type.include_rotary_wing = True
+        parking_type.include_fixed_wing = True
+        parking_type.include_fixed_wing_stol = True
```

Comment:
I see this part returning a lot... perhaps there's a cleaner way to initialize this with the constructor?

Reply:
There actually already was, I was just not using the contructor for that. Personal preference, I guess. I have now changed it as you suggest, and it does look cleaner.